service-registry-cli/README.adoc

```diff
@@ -0,0 +1,327 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-getting-started-rhoas-cli-registry"]
+= Getting started with the Service Registry using rhoas CLI
+:context: getting-started-rhoas-registry
+
+[IMPORTANT]
+====
+The `rhoas` command-line interface (CLI) is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+[role="_abstract"]
+With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal.
+This guide describes how to get started quickly with Service Registry and the RHOAS CLI.
+
+--
+* Using `rhoas` CLI to manage Service Registry instances .
+
+* Retrieve configuration to use Service Registry instance in your application
+
+* Manage Service Registry Artifacts directly from command line
+--
+
+[id="proc-installing-rhoas_{context}"]
+== Installing the rhoas CLI
+
+[role="_abstract"]
+If you haven't installed RHOAS CLI you can install it using the following guide
+ TODO link to CLI installation guide
```

Comment:
Kafka installation guide can be reference here to not replicate steps etc.

Reply:
@PaulRMellor Need your take on this. Do we split tutorials for cli to:

- Installation and login guide
- Getting started with Kafka and RHOAS CLI
- Getting started with Service Regitstry and RHOAS CLI

WDYT?

Reply:
Done that already so we will have look on how this will look like

Reply:
For this release, we can replicate the install and login content. Post-release we can look at how we might split out and have the install as a pre-req.

Reply:
What release? This sounds like duplication of work and reduced user experience.

If I want to read about rhosak for cli I do not want to scroll thru 3 pages of different installation instructions for operating system

Another reason for that is that we should not rename guides that exist and the one is there is cli guide (not kafka guide)